.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - name: Build the tagged Docker image
+        run: docker build --rm --no-cache -t uwpokerclub/app:${GITHUB_REF##*/} -t uwpokerclub/app:latest .
+      - name: Push the tagged Docker image
+        run: docker push uwpokerclub/app:${GITHUB_REF##*/}

```

Comment:
What are the extra # for at the end?

Reply:
Think its because the release ref has extra information, This gets the tag we want https://github.community/t/getting-release-details-from-within-an-actions-run/17552

Reply:
Ah okay cool